sample/src/main/java/io/bladewallet/kotlin_sdk_demo/FirstFragment.kt

```diff
@@ -63,6 +63,47 @@ class FirstFragment : Fragment() {
         }
 
         binding.buttonSecond.setOnClickListener {
+
+//        // QUOTES SWAP
```

Comment:
Is it needed?

Reply:
its part of examples, like other commented code in sample app. in future it will be organised to demo app (WWT-3444)